packages/requests/index.mjs

```diff
@@ -2,17 +2,23 @@ import { AutocannonCLI } from '@expressjs/perf-autocannon';
 import { Wrk2CLI } from '@expressjs/perf-wrk2';
 
 export function startLoad (opts = {}) {
-  const requests = opts.requests || [{
+  let requests = opts.requests || [{
```

Comment:
Keep as `const`, it shouldn't be reassigned.

Reply:
It is re-assigned on line 20. The UX otherwise would be to not support our current sets of requests without `--parallel` being passed and this is an intentionally temporary workaround until we decide which way to go on that (maybe not even support them) but will be a follow up PR. Leaving as is for now, but will address this when we sort out that decision.